src/resources/app/components/AnnouncementCard.vue

```diff
@@ -85,7 +86,7 @@
 
 <script>
 import AnnouncementContent from "./AnnouncementContent";
-import {createComment} from "../../api/announcement/comment/createComment";
+import {createComment} from "../../api/comment";
```

Comment:
```suggestion
import { createComment } from "../../api/comment";
```
spaces between curly brackets